src/GpuLidarSensor.cc

```diff
@@ -129,6 +132,27 @@ bool GpuLidarSensor::Load(const sdf::Sensor &_sdf)
     return false;
   }
 
+  if (_sdf.Element()->HasElement("scanning_pattern"))
+  {
+    auto pattern = _sdf.Element()->Get<std::string>("scanning_pattern");\
```

Comment:
Sorry I just noticed this. There is `\` at the end of the line. If this is needed and compiles, it's fine to leave it. Otherwise it can be deleted.

Reply:
Fixed.